organization-service/src/main/resources/application.yml

```
@@ -1,25 +1,56 @@
+  #    port: 10100
+  endpoints:
+    web:
+      exposure:
+        include: prometheus, metrics
+  metrics:
+    distribution:
+      percentiles-histogram:
+        http.server.requests: true
+  endpoint:
+    prometheus:
+      access: unrestricted
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)